docs/features/kubernetes/configuration.md

```diff
@@ -36,6 +36,7 @@ kubernetes:
             - group: 'argoproj.io'
               apiVersion: 'v1alpha1'
               plural: 'rollouts'
+          customResourceProfile: build # matches with collection name provided in kubernetes.customResourceProfiles
```

Comment:
The config system already has something built-in which has a bit of overlap with this, which is the [`$include` loader](https://backstage.io/docs/conf/writing#including-files). You can point it to data at a path in an external file too, which lets you do something like having a separate `custom-resource-configurations.yaml` where you define these profiles, and then import the ones you need for each cluster. Then again that only works for clusters defined in config, but if they're not defined in config the conciseness is less important.

Seeing as this exists I'm leaning towards using that rather than adding this system on top. Curious what @backstage/kubernetes-maintainers think though.